Language/Variables/Data Types/float.adoc

```diff
@@ -75,6 +69,41 @@ z = (float)x / 2.0; // z now contains .5 (you have to use 2.0, not 2)
 [#see_also]
 --
 
+[float]
+=== Notes and Warnings
+
```

Comment:
```suggestion
If doing math with floats, you need to add a decimal point, otherwise it will be treated as an int. See the link:../../constants/floatingpointconstants[Floating point] constants page for details.

```